.github/workflows/release.yml

```
@@ -3,6 +3,8 @@ name: Release
 on:
   push:
     branches: [ main ]
+    paths-ignore:
+        - 'CHANGELOG.md'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)